nemo/collections/tts/modules/radtts.py

```diff
@@ -60,11 +59,11 @@ def pad_energy_avg_and_f0(energy_avg, f0, max_out_len):
 
 def adjust_f0(f0, f0_mean, f0_std, vmask_bool):
     if f0_mean > 0.0:
-        f0_mu, f0_sigma = f0[vmask_bool].mean(), f0[vmask_bool].std()
-        f0[vmask_bool] = (f0[vmask_bool] - f0_mu) / f0_sigma
+        f0_sigma, f0_mu = torch.std_mean(f0[vmask_bool])
+        f0 = ((f0 - f0_mu) / f0_sigma).to(dtype=f0.dtype)
         f0_std = f0_std if f0_std > 0 else f0_sigma
```

Comment:
is it possible the `f0_sigma` tends to zero? If so, better to take care of zero-division.

Reply:
nit: not quite sure if we need to bother `.to(dtype=f0.dtype)` because all `f0_sigma` and `f0_mu` are computed directly from `f0`. So same dtype should be automatically cast.

Reply:
Well I had errors reported by exporter, that's why I had to add those casts. Division is fp32 when operands are fp16.

Reply:
f0_std was not a problem before, don't see why I should take care of it now.